diff --git a/WinPort/src/Backend/WinPortMain.cpp b/WinPort/src/Backend/WinPortMain.cpp
--- a/WinPort/src/Backend/WinPortMain.cpp
+++ b/WinPort/src/Backend/WinPortMain.cpp
@@ -235,7 +235,7 @@ struct ArgOptions
 	const char *nodetect = "";
 	bool tty = false, far2l_tty = false, notty = false;
 	bool mortal = false;
-	unsigned int esc_expiration = 0;
+	unsigned int esc_expiration = 100; // by unxed: enable by default
 	std::vector<char *> filtered_argv;
 
 	ArgOptions() = default;
diff --git a/far2l/src/cfg/ConfigSaveLoad.cpp b/far2l/src/cfg/ConfigSaveLoad.cpp
--- a/far2l/src/cfg/ConfigSaveLoad.cpp
+++ b/far2l/src/cfg/ConfigSaveLoad.cpp
@@ -265,7 +265,7 @@ public:
 	{true,  NSecInterface, "ExclusiveWinLeft", &Opt.ExclusiveWinLeft, 0},
 	{true,  NSecInterface, "ExclusiveWinRight", &Opt.ExclusiveWinRight, 0},
 
-	{true,  NSecInterface, "OSC52ClipSet", &Opt.OSC52ClipSet, 0},
+	{true,  NSecInterface, "OSC52ClipSet", &Opt.OSC52ClipSet, 1}, // by unxed: enable by default
 
 	{false, NSecInterface, "ShowTimeoutDelFiles", &Opt.ShowTimeoutDelFiles, 50},
 	{false, NSecInterface, "ShowTimeoutDACLFiles", &Opt.ShowTimeoutDACLFiles, 50},
